packages/block-library/src/post-navigation-link/edit.js

```diff
@@ -94,20 +95,28 @@ export default function PostNavigationLinkEdit( {
 	return (
 		<>
 			<InspectorControls>
-				<PanelBody>
-					<ToggleControl
-						__nextHasNoMarginBottom
+				<ToolsPanel label={ __( 'Settings' ) }>
+					<ToolsPanelItem
```

Comment:
All controls are expected to be visible by default for now. Let's add `isShownByDefault` prop to all `ToolsPanelItem` components.